src/Admin/AbstractAdmin.php

```diff
@@ -2676,9 +2694,10 @@ protected function configureDefaultFilterValues(array &$filterValues): void
      *   $sortValues['_sort_by'] = 'foo'
      *   $sortValues['_sort_order'] = 'DESC'
      *
+     * @param array{_page?: int, _per_page?: int, _sort_by?: string, _sort_order?: string} $sortValues*
```

Comment:
The detail is already present in phpstan-param, so we can remove this.

Or use array<string, string|int> which has a better integration by Phpstorm.